loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -453,21 +454,76 @@ export class GLTFLoader implements IGLTFLoader {
         this.log(GLTFLoaderState[this._state]);
     }
 
-    private _createRootNode(): INode {
-        this._rootBabylonMesh = new Mesh("__root__", this._babylonScene);
-        this._rootBabylonMesh.setEnabled(false);
+    private _checkForRootNode(): Nullable<INode>{
+        if (this._gltf.scenes == null) {
+            return null;
+        }
 
-        const rootNode: INode = {
-            _babylonTransformNode: this._rootBabylonMesh,
-            index: -1
-        };
+        var indexScene = this._gltf.scene || 0;
+        var scene = this._gltf.scenes[indexScene];
+
+        // Root node must be unique
+        if (scene.nodes.length != 1) {
+            return null;
+        }
+        else if(this._gltf.nodes != null) {
+            var indexRoot = scene.nodes[0];
+            var root = this._gltf.nodes[indexRoot];
+
+            // Check rotation and scale
+            if(root.rotation != undefined && root.scale != undefined){
+                if (root.rotation.join(',') != "0,0,0,1" || root.scale.join(',') != "1,1,-1") {
+                    return null;
+                }
+            }
+
+            // Check name
+            if (root.name != "__root__") {
```

Comment:
not sure we want to check this one. As long as we have the correct transforms we are fine